data/json/items/comestibles/junkfood.json

```
@@ -1290,5 +1290,21 @@
     "flags": [ "EATEN_HOT" ],
     "vitamins": [ [ "vitA", 3 ], [ "iron", 10 ] ],
     "fun": 4
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "chocolate_mre",
+    "name": { "str": "military chocolate bar" },
+    "copy_from": "chocolate",
+    "calories": 1800,
+    "weight": "115 g",
+    "description": "A thick, dense bar of military chocolate.  While tough to chew and not very appetizing, it will provide a day's worth of calories in portable form.",
```

```suggestion
    "description": "A thick, dense bar of military chocolate.  While tough to chew and not very appetizing, it will provide almost a day's worth of calories in portable form.",
```
1800 is below the standard 2000 (and both military and survivors tend to use up more than that average!).